commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountUserRelLocalServiceImpl.java

```diff
@@ -134,6 +144,49 @@ public void addCommerceAccountUserRels(
 		}
 	}
 
+	@Override
+	public void addDefaultRole(long userId, ServiceContext serviceContext)
+		throws PortalException {
+
+		User user = userPersistence.findByPrimaryKey(userId);
+
+		Set<Long> roleIdSet = new HashSet<>();
```

Comment:
Why do we need this `Set`? It looks like it only ever has 1 entry, and it's only used to create an array, so we might as well just directly create an array of 1, yeah?